packages/itwin/property-grid/src/components/FilteringPropertyGrid.tsx

```diff
@@ -71,11 +77,11 @@ export class NonEmptyValuesPropertyDataFilterer extends PropertyRecordDataFilter
 }
 
 type CustomPropertyDataProvider<TPropertyData> = IDisposable &
-Omit<IPropertyDataProvider, "getData"> & {
-  getData: () => Promise<TPropertyData>;
-};
+  Omit<IPropertyDataProvider, "getData"> & {
```

Comment:
these indentations are probably gonna fail the linter

Reply:
Ran the linter and fixed issues.